mattumland/cloudbank
# Feature/names

## Is this a fix or a feature?
Feature
## What is the change?
Adds fetch for random names from the star trek API
## What does it fix?

## Where should the reviewer start?
apiCaller.js
## How should this be tested?
Load then reload the page multiple times while watching the console to confirm the names are showing and changing.
